tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -256,7 +265,9 @@ export class PodmanDesktopRunner {
     }
 
     try {
-      if (test.info().status === 'failed') return;
+      const testStatus = test.info().status;
+      console.log(`Test finished with status:${testStatus}`);
+      if (testStatus === 'passed' || testStatus === 'skipped') return;
```

Comment:
```suggestion
      if (testStatus !== 'passed' && testStatus !== 'skipped') return;
```

Shouldn't it be like this? If the test iksn't passed and isn't skipped then we can proceed with the trace/video deletion.

Reply:
oh, good catch!

Reply:
fixed